keras_cv/layers/preprocessing/rgb_shift.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np 
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class RGBShift(layers.Layer):
+    """RGBShift class randomly shift values for each channel of the 
+    input RGB image. The expected images should be `(0-255)` pixel ranges.
+
+    Input shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
+    Output shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
```

Comment:
@fchollet Should I remove lines 24-29 in a favor of `Call argument` docstring in lines 38-44.

Reply:
These lines are fine, we can keep them.